docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1165,6 +1165,10 @@ to the connector using a :doc:`WITH </sql/create-table-as>` clause::
       ``s3a://test/name=${name}/``. Mapped from the AWS Athena table property
       `storage.location.template <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-setting-up.html#partition-projection-specifying-custom-s3-storage-locations>`_
     - ``${table_location}/${partition_name}``
+  * - ``extra_properties``
+    - Additional properties added to a Hive table. The properties are not used by Trino,
+      and are not shown in SHOW CREATE TABLE statements.
```

Comment:
currently not shown (don't promise they won't be shown in the future)

Reply:
... can be seen in $properties

Reply:
@mosabua Can you review the document changes.